muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/dropdown.tsx

```diff
@@ -563,7 +567,11 @@ export default class Dropdown extends React.Component<
             }}
           ></span>
           {(this.props.content || this.props.items) && (
-            <div className="dropdown__container">
+            // We use tooltipId prop here so we can attach tooltip's trigger button's [aria-describedby] attibute directly to the content of the tooltip
+            <div
+              className="dropdown__container"
+              id={this.props.tooltipId ? this.props.tooltipId : ""}
+            >
```

Comment:
Conditional checking is not necessary as id attribute is not required anyways for divs and empty string doesn't do anything